pyfolio/timeseries.py

```diff
@@ -580,8 +584,27 @@ def rolling_fama_french(returns, factor_returns=None,
         factor_returns = factor_returns.drop(['Mkt-RF', 'RF'],
                                              axis='columns')
 
-    return rolling_beta(returns, factor_returns,
-                        rolling_window=rolling_window)
+    regression_df = pd.concat([returns, factor_returns], axis='columns')
+    regression_df.columns = ['rets', 'SMB', 'HML', 'UMD']
+
+    # have NaNs when there is insufficient data to do a regression
+    regression_coeffs = np.empty((len(regression_df.index[:rolling_window]),
+                                  len(factor_returns.columns)))
+    regression_coeffs.fill(np.nan)
+
+    for beg, end in zip(regression_df.index[:-rolling_window],
+                        regression_df.index[rolling_window:]):
+        window = regression_df.loc[beg:end]
+        coeffs = sm.ols(formula='rets ~ SMB + HML + UMD - 1', data=window) \
+            .fit().params.values
```

Comment:
I also want to make sure that this computation is correct. Are these parameters the Fama French betas?

The `-1` in the `formula` keyword means to set the intercept equal to 0.